src/Exception/FrozenAttributeException.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use Rinq\Ident\Attribute;
+use RuntimeException;
+
+class FrozenAttributeException extends RuntimeException
```

Comment:
Needs a doc-block, as this is part of the public interface, something like:
> Indicates a failure to update a revision because one or more of the updated attributes are frozen.